### PR TITLE
fixed contrast ratio, accessibility issues

### PR DESCRIPTION
**Pull request by anishagg17**
**Description**

fixed contrast ratio, accessibility issues

**Technical**

fixes contrast ratio in language-select by using a more brighter colour
fixes accessibility issue in iajs component by adding aria-lable

**Evidence**

Screenshot 2020-03-08 at 3 30 23 PM

Screenshot 2020-03-08 at 3 29 50 PM

### CODE REVIEW

File: packages/ia-components/sandbox/iajs-test-component/index.jsx [hunk 1]
```diff
@@ -32,17 +32,18 @@ export default class extends Component {
...
   }
 
-  render () {
+  render() {
     return (
       <div className="iajs-test-component">
         <h5>My identifier is {this.props.identifier}</h5>
         <img
-          style={{maxWidth: '300px', height: 'auto'}}
+          aria-label="iajs-test-component"
```
Comment (shaneriley) [inline packages/ia-components/sandbox/iajs-test-component/index.jsx]: This image is entirely decorative, and as such it makes more sense to simply add an empty `alt` attribute to conform to a11y standards as described in https://www.w3.org/WAI/tutorials/images/decorative/

File: packages/ia-components/sandbox/language-select/language-select.less [hunk 2]
```diff
@@ -1,5 +1,5 @@
 @dropdown_background_color: #222222;
-@text_color: #848484;
+@text_color: #b9b9b9;
```
Comment (shaneriley) [inline packages/ia-components/sandbox/language-select/language-select.less]: I've put in a request to have this color change approved by our design team before approving. Will let you know with a follow-up comment if this change is acceptable.

File: packages/ia-components/sandbox/iajs-test-component/index.jsx (hunk 1 above)
Comment (shaneriley) [reply packages/ia-components/sandbox/iajs-test-component/index.jsx]: The alt attribute should be empty, not populated with the parent div's class value.

File: packages/ia-components/sandbox/language-select/language-select.less (hunk 2 above)
Comment (shaneriley) [reply packages/ia-components/sandbox/language-select/language-select.less]: Ok, from the design team: "We've been moving to #999999 for these. It's AA compliant for small text against the #222 background, and it's easier to differentiate from the #fff active text color." Updating this to #999999 and setting the image alt attribute to "" will make this ready to merge.

**Comment by anishagg17** [comment]
@shaneriley I have made the requested changes

**Comment by anishagg17** [comment]
@shaneriley I have made the requested changes

**Pull request closed without merging**